app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelChairAccessPublicTransport.java

```diff
@@ -19,7 +19,8 @@
 	@Override protected String getTagFilters()
 	{
 		return " nodes, ways, relations with (public_transport=platform or " +
-				"(highway=bus_stop and public_transport!=stop_position))" +
+				"(highway=bus_stop and public_transport!=stop_position) or " +
+				" railway ~ station|subway_entrance)" +
```

Comment:
Oh! Looks like I removed this feature mid of January. Oh well, I guess back then I thought "elements" would be too intransparent.